packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -672,6 +672,37 @@ void main() {
       final Offset scenePoint = transformationController.toScene(localFocalPoint);
       expect(scenePoint, const Offset(100, 100));
     });
+    
+     testWidgets('Scaling amount is equal forth and back with a mouse scroll', (WidgetTester tester) async {
+      final TransformationController transformationController = TransformationController();
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+              body: Center(
+            child: InteractiveViewer(
+              constrained: false,
+              maxScale: 100000,
+              minScale: 0.01,
+              transformationController: transformationController,
+              child: Container(width: 1000.0, height: 1000.0),
+            ),
+          )),
+        ),
+      );
+
+      final Offset center = tester.getCenter(find.byType(InteractiveViewer));
+      await scrollAt(center, tester, const Offset(0.0, -200.0));
+      await tester.pumpAndSettle();
+      expect(transformationController.value.getMaxScaleOnAxis(), math.exp(200 / 200));
+      await scrollAt(center, tester, const Offset(0.0, -200.0));
+      await tester.pumpAndSettle();
+      // math.exp round the number too short compared to the one in transformationController.
+      expect(double.parse((transformationController.value.getMaxScaleOnAxis()).toStringAsFixed(15)), math.exp(400 / 200));
```

Comment:
I think you can use [closeTo](https://master-api.flutter.dev/flutter/package-matcher_matcher/closeTo.html) here instead of converting to String.

Reply:
I put the max digit after 0 just before it break